src/__tests__/update-logs.test.ts

```
@@ -0,0 +1,352 @@
+				context: 'test_context',
+				source: 'ratos-update'
+			});
+			expect(logEntry).toHaveProperty('time');
+			expect(logEntry).toHaveProperty('pid');
+			expect(logEntry).toHaveProperty('hostname');
+		} finally {
+			// Restore original environment
+			if (originalLogFile) {
+				process.env.RATOS_LOG_FILE = originalLogFile;
+			} else {
+				delete process.env.RATOS_LOG_FILE;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)